pkg/api/k8s_versions.go

```diff
@@ -19,7 +19,7 @@ const (
 	calicoCNIImageReference                           string = "cni:v3.8.0"
 	calicoNodeImageReference                          string = "node:v3.8.0"
 	calicoPod2DaemonImageReference                    string = "pod2daemon-flexvol:v3.8.0"
-	calicoClusterProportionalAutoscalerImageReference string = "cluster-proportional-autoscaler-amd64:1.1.2-r2"
+	calicoClusterProportionalAutoscalerImageReference string = "oss/kubernetes/autoscaler/cluster-proportional-autoscaler:1.1.2-r2"
```

Comment:
I think we need to turn this into the full mcr URL, and get rid of the `kubernetesImageBase` prefix dep. Basically we don't need to support both GCR and MCR types here.

Also, not to add scope creep, but is it worth seeing if a newer version of this works w/ the current calico spec?

Reply:
> is it worth seeing if a newer version of this works...?

I think so. I added a commit to put all three versions of C-P-A in the VHD but ideally we would all use 1.7.1 (or the 1.7.1-hotfix?). I'll try that next.

Reply:
Actually, since this is a new regression in v0.49.0, let's defer the version bumps to a separate PR.